en/core-api/e-wallet.md

```diff
@@ -1,76 +1,75 @@
 
-GoPay is an e-Wallet payment method by Gojek. Users will pay using the Gojek apps, or any QRIS compatible app. The user flow varies when using a web browser (on a computer or a tablet) compared to a SmartPhone:
+Gojek has introduced GoPay as an E-Wallet payment method. Customers can pay using the Gojek apps, or any QRIS compatible app.  The user flow is different for web browser (on a computer or a tablet) and smartphone. The user flows are given below.
 
-1. **QR Code** - This is the user flow on a web browser (on a computer or a tablet). User is shown a QR code and asked to scan using any QRIS compatible app, like Gojek app.
-2. **Deeplink** - This is the user flow on a SmartPhone/mobile device. User gets redirected to the Gojek apps to finish payment.
+1. **QR Code** - This is the user flow on a web browser (on a computer or a tablet). User is shown a QR code and asked to scan using any QRIS compatible app, such as Gojek app.
+2. **Deeplink** - This is the user flow on a smartphone /mobile device. User gets redirected to the Gojek app to finish payment.
 
-Basic integration process of GoPay will be explained below.
-
-?> Please make sure you have already done [creating your Midtrans Account](/en/midtrans_account/overview.md), before proceeding with this section.
-
-## Integration Step
-1. Send transaction data to API Charge.
-2. Alter payment flow depending on the device used.
-3. Handling Post-Transaction.
+?> Please make sure to create your[ Midtrans Account](/en/midtrans_account/overview.md), before proceeding with this section.
 
 <details>
 <summary><b>Sequence Diagram</b></summary>
 <article>
 The overall GoPay end-to-end payment proccess can be illustrated in following sequence diagram:
+<!--spelling of process. colon after the description in the description of sequence diagram.-->
 
 <!-- tabs:start -->
+
 #### **QR Code Mode (Default)**
+
 ![Core API Gopay QR sequence diagram](./../../asset/image/core-api_sequence_qr.png)
 
 #### **Deeplink Mode**
+
 ![Core API Gopay Deeplink sequence diagram](./../../asset/image/core-api_sequence_deeplink.png)
 <!-- tabs:end -->
 </article>
 </details>
 
-## 1. Send Transaction Data to API Charge
-API request should be done from **Merchant’s backend** to acquire qr code and deeplink to Gojek app. There are several components that are required:
+## Steps for Integration
 
-Requirement | Description |
------------ | ----------- |
-Server Key | Explained on [previous section](/en/midtrans-account/overview.md) |
-`order_id` | Transaction order ID, defined from your side |
-`gross_amount` | Total amount of transaction, defined from your side |
-`payment_type` | Set to `gopay` |
+Basic integration process of GoPay will be explained in this section. To integrate with *E-Wallet* payment  method, follow the steps given below.
 
-Charge API request should be done from Merchant's backend.
+### 1. Sending Transaction Data to API Charge
```

Comment:
The page structure is messed up, please be mindful with the header level. Each header level `#`, `##`, `###` is meaningful, do not change the level if you do not understand it.
<img width="265" alt="Screen Shot 2020-11-03 at 17 04 28" src="https://user-images.githubusercontent.com/13027142/97972266-d6edc880-1df6-11eb-838a-361846855f5f.png">